### DISCUSSION TITLE
Dashboard asking donation in large information bar

### DISCUSSION THREAD

**Discussion by FreyGabor1985** [AzuraCast/AzuraCast#6977] [General]
Hi,
There is any way to disable the information bar what's pops up always and asking donation? 
I don't want and will never remove azuracast branding also shows on my back and public pages, but this bar using the screen space and starting a bit annoying me. I can understand the author wants the credit but donation option already in the footer on every page and visible always and now is on the dashboard also with the large bar. Im just started to use azuracast and Im happy with and Im also happy to donate as well but this is just a too much force and push us to donate. I belive if I start to donate then the bar will still there and keeps me annoying. :( 
I kindly ask there any way to remove this bar? Not the branding or footer...etc. just this info bar....
Many thanks
<img width="1053" alt="Screenshot 2024-02-27 at 10 00 33" src="https://github.com/AzuraCast/AzuraCast/assets/161453491/b763fcf0-0c7b-4638-92ae-61df19b328b7">

**Comment by Heimatradio** [comment]
I'm at a loss for words: You can also rent a streaming server and then buy an expensive tool to manage it. This massive software 'AzuraCast' is being developed with a lot of work. Then scroll down a few millimeters and you won't see it.

**Reply by FreyGabor1985** [reply]
I think you misunderstand what I was asking. Can you read again my question? 
"Not the branding or footer...etc. just this info bar...."
I don't mind to see and show the footer donations, info, credit...etc.... But this info bar is unnecessary in the dashboard. I think I didn't says or requested anything wrong and it's not harm anyone due all credits and donation option is still visible and available. 
I do respect all the coding spent time from the author but this info bar is just too much force to donation. 
I also happy to make a monthly donation if this bar removable. I think is fair and acceptable. :) Thats All I was asking.

**Comment by BusterNeece** [comment]
I don't plan on officially adding a way to hide this notification. It only comes up once in a while and deliberately doesn't come up on repeat requests or in development environments.

If you insist upon removing it, you can do so via custom CSS:

```css
[data-bs-theme] #dashboard .alert-info {
  display: none;
}
```
